packages/main/src/plugin/kubernetes/kubernetes-client.ts

```diff
@@ -948,7 +948,7 @@ export class KubernetesClient {
         plural: 'routes',
         name,
       });
-      const route = res?.body as V1Route;
+      const route = res as V1Route;
```

Comment:
question: how do we check this through a unit test ?

Reply:
This would depend on what the library returns, meaning that we would need some kind of integration tests, with at least the library